db_schema/readrsql.patch.6.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE comments ( id BIGINT UNSIGNED PRIMARY KEY AUTO_INCREMENT, body varchar(500), og_title varchar(256), og_description varchar(256), og_image varchar(128), like_amount int, parent_id bigint UNSIGNED, resource varchar(128), created_at datetime, updated_at datetime, author BIGINT UNSIGNED, active tinyint, status tinyint, ip int unsigned);
```

Comment:
ip int unsigned?

Reply:
According to this post: 
https://stackoverflow.com/questions/5133580/which-mysql-datatype-to-use-for-an-ip-address?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Store it as unsigned int is an option.
Or should I use string datatype to store IP value?

Reply:
Just not sure which option is better.

Reply:
ok, unsigned int could be faster in data access.